README.md

```diff
@@ -109,7 +109,7 @@ The following resources will help you get started. For more information, check o
 1. [Wiki](https://github.com/Microsoft/WinObjC/wiki), for documentation and tutorials
 2. [Development Roadmap](https://github.com/Microsoft/WinObjC/wiki/Roadmap), detailing our highest priorities
 3. [App Analysis Tool](https://developer.microsoft.com/en-us/windows/bridges/ios/app-analyzer-tool), to evaluate the compatibility of your app with the bridge
-4. [Website on Windows Dev Center](https://dev.windows.com/bridges/ios), for evaluation virtual machines
+4. [Website on Windows Dev Center](https://dev.windows.com/bridges/ios), for evaluation of virtual machines
```

Comment:
This should still say "for evaluation virtual machines"; the VMs are "evaluation VMs" :smile: